examples/cinnabar-api.ipynb

```diff
@@ -0,0 +1,1817 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.            * unit.kilocalorie_per_mole,  # the uncertainty in the calculated relative free energy with units
</pre><p>If possible, don't break this over two lines - it's not PEP8 friendly, but from a reader perspective having it broken over two lines is worse.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/cinnabar/pull/160/discussion/#comment-2490530734'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>